doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3685,6 +3685,11 @@ Setting implicit automation tactics
       time the term argument of a tactic has one of its holes not fully
       resolved.
 
+      .. deprecated:: 8.9
+
+         This command is deprecated. Use :ref:`typeclasses <typeclasses>` or
+         :ref:`tactics-in-terms <tactics-in-terms>`instead.
+
    .. example::
 
       .. coqtop:: all
```

Comment:
You did as I suggested and you've successfully managed to move this Sphinx warning a few lines down to the part that you didn't actually touched. Unfortunately, this `.. example::` and the inner `.. coqtop:: all` have to be similarly re-indented to be clearly on the same level as the preceding `.. deprecated::` and the paragraph above, inside the `.. cmd::` body. I am sorry that you are paying the price of a badly handled Sphinx migration :confused:

Reply:
Gaaah...